scipy/sparse/linalg/dsolve/linsolve.py

```diff
@@ -456,6 +457,9 @@ def spsolve_triangular(A, b, lower=True, overwrite_A=False, overwrite_b=False):
     lower : bool, optional
         Whether `A` is a lower or upper triangular matrix.
         Default is lower triangular matrix.
+    unit_diagonal : bool, optional
+        If True, diagonal elements of `a` are assumed to be 1 and will not be
+        referenced.
```

Comment:
`versionadded` directive might be in order?

Reply:
added